adafruit_midi/note_off.py

```diff
@@ -38,17 +38,17 @@ class NoteOff(MIDIMessage):  # pylint: disable=duplicate-code
     def __init__(self, note, velocity=0, *, channel=None):
         self.note = note_parser(note)
         """key, either int (0-127) or string that will be turned off """
-        self._velocity = velocity
+        self.velocity = velocity
```

Comment:
Please add a comment after it like self.note above. That will make it show up in the docs.